doc/docs/Python_User_Interface.md

```diff
@@ -1471,7 +1471,7 @@ Once you have called `add_energy`, the Fourier transforms of the fields are accu
 <div class="class_members" markdown="1">
 
 ```python
-def display_electric_energy(self, *energys): 
+def display_electric_energy(self):
```

Comment:
Again, `*energys` should not be removed from the function header. Same for the related functions `display_magnetic_energy` and `display_total_energy`.